README.md

```diff
@@ -28,10 +28,7 @@ $ npm install @vtex/delivery-packages
 ```js
 const packagify = require('@vtex/delivery-packages')
 
-packagify({
-  items,
-  selectedAddresses,
-  logisticsInfo
+packagify(order)
 })
 // [
```

Comment:
Comments?

Reply:
That's an example of the output, like this:
https://github.com/sindresorhus/query-string#usage